utils/api/requests.js

```diff
@@ -51,22 +52,27 @@ export const useAllSOWs = (id, requestIdentifier, accessToken) => {
   }
 }
 
-export const useAllMessages = (id, accessToken) => {
+export const useMessagesAndFiles = (id, accessToken) => {
   const { data, error, mutate } = useSWR(id ? [`/quote_groups/${id}/notes.json`, accessToken] : null)
   let messages
-  if (data) messages = configureMessages(data.notes)
+  let files
+  if (data) {
+    messages = configureMessages(data.notes)
+    files =  configureFiles(data.notes)
+  }
 
   return {
     data,
     messages,
+    files,
     mutate,
-    isLoadingMessages: !error && !data,
-    isMessageError: error,
+    isLoadingMessagesAndFiles: !error && !data,
+    isMessagesAndFilesError: error,
   }
 }
 
 // TODO(alishaevn): refactor the below once the direction of https://github.com/scientist-softserv/webstore/issues/156 has been decided
-export const postMessageOrAttachment = ({ id, message, files, accessToken }) => {
+export const postMessageOrFile = ({ id, message, files, accessToken }) => {
```

Comment:
```suggestion
export const createMessageOrFile = ({ id, message, files, accessToken }) => {
```

Reply:
suggesting so we have some uniformity in our function naming.
GET => `use...`
POST => `create...`

the name would also need to updated in all the places where it's used.

Reply:
updated 👍